### ISSUE TITLE
ReferenceError: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by ForbesLindesay** [codemix/flow-runtime#95]
This is a:

 - [x] Bug Report
 - [ ] Feature Request
 - [ ] Question
 - [ ] Other

Which concerns:

 - [x] flow-runtime
 - [ ] babel-plugin-flow-runtime
 - [ ] flow-runtime-validators
 - [ ] flow-runtime-mobx
 - [ ] flow-config-parser
 - [ ] The documentation website

---

### What is the current behaviour?

When I run code compiled with regenerator runtime, it throws "ReferenceError: regeneratorRuntime is not defined"

---

### What is the expected behaviour?

It should not throw an error.
---

### Which package versions are you using?

0.10.0

I'm using it in a browserify bundle.

My guess is that flow-runtime needs to be compiled with the babel-plugin-transform-runtime package.

**Comment by vicapow** [comment]
see: https://github.com/codemix/flow-runtime/issues/35

tl;dr, add  `import 'regenerator-runtime/runtime';` somewhere before flow-runtime gets imported

**Comment by phpnode** [comment]
Thanks, yes sorry this is a dupe of #35, I misread it. I don't want to include regenerator runtime directly because any projects that use generators will already be including it and it won't get deduped, it's a bit unfortunate and really I'd rather just ship the generators, but that causes even more tooling problems.
